config/config.yaml

```diff
@@ -3,28 +3,23 @@ apps:
     port: 3000
     privatePort: 4000
     useCachingInterceptor: true
-  cacheWarmer:
-    port: 4001
-  queueWorker:
-    port: 4002
-  transactionsProcessor:
-    port: 4003
-    maxLookBehind: 100
 libs:
   common:
+    governance:
+      contractAddress: 'erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqrlllsrujgla'
```

Comment:
maybe it better to add the contract address in .env?

Reply:
It is a static configuration (a System Smart Contract)

Reply:
got it